packages/gatsby/src/redux/persist.ts

```diff
@@ -81,10 +81,16 @@ function guessSafeChunkSize(values: [string, IGatsbyNode][]): number {
   const valueCount = values.length
   const step = Math.max(1, Math.ceil(valueCount / nodesToTest))
   let maxSize = 0
+  
```

Comment:
It's a minor thing, but could you please drop this new line as well?

Generally you'd want to create as small a diff as possible. In this case this empty line is not related to this PR so it's best to drop those kinds of changes before submitting a patch / PR.
- The fewer changes in a PR the easier it is for a reviewer to review (less cognitive overhead)
- Unnecessary changes affect the git history, which in turn unnecessarily affects bisecting problems and `git blame`

In that respect, the new empty line 80 is fine since it's part of your patch.